components/compliance-service/ingest/pipeline/processor/parse_struct.go

```diff
@@ -28,11 +28,22 @@ func MapStructsESInSpecReportToControls(inspecReport *relaxting.ESInSpecReport)
 	nodes := make([]relaxting.Node, 0)
 	// Get the nodes
 	node := relaxting.Node{NodeUUID: inspecReport.NodeID,
-		DailyLatest: inspecReport.DailyLatest,
-		DayLatest:   inspecReport.DayLatest,
-		NodeEndTime: inspecReport.EndTime,
-		Status:      inspecReport.Status,
-		ReportUUID:  inspecReport.ReportID}
+		DailyLatest:      true,
+		DayLatest:        true,
+		NodeEndTime:      inspecReport.EndTime,
+		Status:           inspecReport.Status,
+		ReportUUID:       inspecReport.ReportID,
+		NodeName:         inspecReport.NodeName,
```

Comment:
we already have unit tests for existing fields, we need to enhance them to test newly added fields.

Reply:
Added test cases for new fields